src/web/resources/report-page.js

```diff
@@ -153,11 +153,14 @@ function update(jsonData) {
 }
 
 // View State
-var detailsState = false
+var filterDetailsState = false
 var groupState = {
     "improved": true,
     "regressed": true
 }
+var benchMarkState = "All"
+var benchMarks = ["All", "Mathematics", "Hamming", "Nightly", "Libraries", "Numerics", "Physics", "Demo", "Haskell", "Pbrt", "Regression", "Tutorial"]
```

Comment:
I would generate `benchMarks` from the data. And I would use `false` as the special "all" state not the word `"all"`